snapcraft/linters/library_linter.py

```diff
@@ -101,6 +104,20 @@ def run(self) -> List[LinterIssue]:
 
         return issues
 
+    def _find_deb_package(self, library_name):
+        for path in glob.glob("/usr/lib/**", recursive=True):
+            if os.path.basename(path) == library_name:
+                try:
+                    output = subprocess.run(
+                        ["dpkg", "-S", path], check=False, stdout=subprocess.PIPE
+                    )
+                except:  # pylint: disable=bare-except
```

Comment:
we should be capturing specific exceptions

Reply:
Fixed.